### ISSUE TITLE
No `nest.help()` when installed through Conda

### ISSUE THREAD

**Issue by terhorstd** [nest/nest-simulator#2483]
The NEST help doesn't work when installed through conda. So `nest.help('iaf_psc_exp')` gives the following error:

    Sorry, there is no help for model 'iaf_psc_exp'.
    Use the Python help() function to obtain help on PyNEST functions.

This should either give useful information, or redirect to online documentation.

**Supporting materials**
This function is mentioned in different workshop and tutorial materials.

**Desktop/Environment**
Various, since it's in a workshop environment potentially with own systems.

**Comment by steffengraber** [comment]
Played around a bit with the installation of NEST. It does not only apply to the conda package. To get this function to work, you still need to run a `make html` during the installation:

    make
    make html
    make install

This is not the case in the conda package and, to my knowledge, is not described in the NEST documentation. (?) Should we integrate it in the conda package?

**Comment by steffengraber** [comment]
In the NEST docker image, the installation is carried out in this way, which means that `nest.help('iaf_psc_exp')`  works there.

**Comment by babsey** [comment]
> Should we integrate it in the conda package?

I would like to have it integrated in conda package. :pray:

**Comment by zbarni** [comment]
From my limited experience with getting NEST installed in various ways by various people, installation via conda is still much more common than through docker, for example. I believe that adding it to the conda package would be helpful for many users.

**Comment by steffengraber** [comment]
The problem is that with `make html` the entire user documentation is installed. In other words, a lot of things that are never needed in a normal installation.

**Comment by zbarni** [comment]
Right, maybe one can make this decision based on users' preferences / current usage. However, if it's not done by default, then perhaps the documentation site [here](https://nest-simulator.readthedocs.io/en/v3.3/getting_help.html) should be updated, either by removing the `nest.help('model_name')` entry or making it clear that this only works if the documentation has been built (manually).

**Comment by steffengraber** [comment]
In preparation for NEST 3.4, I have tried to integrate the documentation into the package. Unfortunately, this does not work out of the box. The problem are these lines in conf.py:

```
if os.environ.get("READTHEDOCS") == "True":
    doc_build_dir = source_dir / "doc/htmldoc"
else:
    doc_build_dir = Path(os.environ["OLDPWD"]) / "doc/htmldoc"
```

OLDPWD does not seem to exist in the conda build environment.

**Comment by terhorstd** [comment]
> Right, maybe one can make this decision based on users' preferences / current usage. However, if it's not done by default, then perhaps the documentation site [here](https://nest-simulator.readthedocs.io/en/v3.3/getting_help.html) should be updated, either by removing the `nest.help('model_name')` entry or making it clear that this only works if the documentation has been built (manually).

@jessica-mitchell , could we remove the `nest.help(…)` and put a link to the Model Directory instead? Much of the model documentation would also be available in NEST Desktop through the model details button (which are derived from the online docs afaik).

As discussed in the last days, dropping `nest.help(…)` in favor of online docs seems to be fine. The existing references would need to be updated though. A working Python builtin `help` like `help(nest.iaf_psc_alpha)` would require a bit more work and is rather mid-term. With NESTML and it's model code generation this switch becomes much simpler.

**Comment by steffengraber** [comment]
@terhorstd @jessica-mitchell `IT resources request - CEA (FR) - Interactive Computing Cluster - Interactive Computing Services
Please specify how many core hours are requested, e.g. '1000 CPU core hours + 2000 GPU core hours' if both are needed or '3000 CPU core hours' if only one kind is required. `

I could do it in #2557. What do you mean?
